docs/generate.html

```diff
@@ -610,6 +622,42 @@ <h2 class="mt-3">Location Sharing</h2>
           <option value="none">None</option>
           <option value="intense">Intense: Sharing physical location to other users e.g. a postal address</option>
         </select>
+        <button type="button" onClick="advance_tab()" class="btn btn-info mt-4 float-right">Next</button>
+      </div>
+
+      <!-- Mood pane -->
+      <div class="tab-pane mt-3" id="mood" role="tabpanel">
+
+        <h2 class="mt-3">General intensity</h2>
+        <p>
+          A summary of the general intensity of the content, e.g. the pacing of a piece of media or pressure to act on the player in a videogame.
```

Comment:
is this game specific?

Reply:
It is; I've left the phrasing vague here with a view to people maybe wanting to apply OARS descriptors to other media as well.